### ISSUE TITLE
train_batch_size and time required to pretrain

### ISSUE THREAD

**Issue by Jimojimojimo** [guotong1988/BERT-GPU#29]
When I set train_batch_size to 8 and experiment with 8 GPUs, the batch_size will be 64 overall.
Then I think the learning speed should be faster than when I did batch_size 8 with a single GPU, but when I actually learned it, it takes a similar amount of time.
Is something going wrong?

**Comment by guotong1988** [comment]
Learning rate is the same for 8-batch-size and 64-batch-size.

See the readme.

**Comment by guotong1988** [comment]
![image](https://user-images.githubusercontent.com/4702353/126776473-c5273d43-0087-4974-bd50-5a1c89150f58.png)
